### DISCUSSION TITLE
comit and push files

### DISCUSSION THREAD

**Discussion by ThibeauThomasPXL** [community/community#42280] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I am working in webstorm to create a webiste, but when I want to upload files to github the console says 2 too large files want to push. While these files are not even selected to push. How can I fix this?

**Comment by SilasRodrigues19** [comment]
Hi @ThibeauThomasPXL 

There is a section in the GitHub docs explaining this. Here's the link: 
[https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github)
